pkg/gemspec/parse.go

```diff
@@ -0,0 +1,104 @@
+package gemspec
+
+import (
+	"bufio"
+	"io"
+	"path/filepath"
+	"regexp"
+	"strings"
+	"unicode/utf8"
+
+	"github.com/aquasecurity/go-dep-parser/pkg/types"
+	"golang.org/x/xerrors"
+)
+
+var (
+	bwQuotes         = regexp.MustCompile(`['|"](.*?)['|"]`)
+	gemSpec          = ".gemspec"
+	gemStr           = ".name = "
+	gemStrVer        = ".version = "
+	gemStrLic        = ".licenses = "
+	packageNameRegEx = regexp.MustCompile(`^[a-zA-Z0-9_\-]+$`)
+	versionRegEx     = regexp.MustCompile(`^[0-9.]+$`)
+	fileNameVersion  = regexp.MustCompile(`^([\w_.\-]+)-([0-9.]+)$`)
+)
+
+func Parse(r io.Reader, filePath string) ([]types.Library, error) {
```

Comment:
If it always returns a single library, you don't have to use []types.Library.

```suggestion
func Parse(r io.Reader, filePath string) (types.Library, error) {
```